101-sql-vm-new-storage-ultrassd/azuredeploy.json

```diff
@@ -156,22 +156,20 @@
       "type": "String",
       "defaultValue": "G:\\SQLLog",
       "metadata": {
-        "description": "Path for SQL Log files"
-      }
-    },
-    "tempDbPath": {
-      "type": "String",
-      "defaultValue": "D:\\SQLTemp",
-      "metadata": {
-        "description": "Path for SQL Temp DB files"
+        "description": "Path for SQL Log files. Please choose drive letter from F to Z and different than the one used for SQL data. Drive letter from A to E are reserved for system"
       }
     },
     "location": {
       "type": "string",
       "metadata": {
         "description": "Location for all resources."
       },
-      "defaultValue": "[resourceGroup().location]"
+      "allowedValues": [
+        "East US 2",
+        "SouthEast Asia",
+        "North Europe"
+      ],
+      "defaultValue": "East US 2"
```

Comment:
I assume the location requirements are temporary?  remove the defaultValue and set a value in the parameters file, otherwise the "default" location tends to get overloaded.

Reply:
Removed defaultValue.  My understanding is that Ultra SSD is limited to 3 regions for now due to the hardware they used.  I'll update the template if more it's available in more regions.